Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserRemovedFromServerRole.yaml

```diff
@@ -1,6 +1,9 @@
 id: 5dd79877-8066-4ce4-ae03-eedd8ebf04f8
 name: User removed from SQL Server Roles
 description: |
+     This hunting query identifies user removed from a SQL Server Role.
```

Comment:
"This hunting query identifies user removed from a SQL Server Role. It relies on the SQLEvent KQL Parser function."